application/src/main/java/org/togetherjava/tjbot/features/moderation/scam/ScamBlocker.java

```diff
@@ -251,7 +251,9 @@ private void dmUser(MessageReceivedEvent event) {
     }
 
     private void dmUser(Guild guild, long userId, JDA jda) {
-        jda.openPrivateChannelById(userId).flatMap(channel -> dmUser(guild, channel)).queue();
+        jda.openPrivateChannelById(userId).flatMap(channel -> dmUser(guild, channel)).queue(any -> {
+        }, failure -> logger.debug("Unable to send dm message to user {} in guild {}", userId,
+                guild.getId()));
```

Comment:
dont silently ignore the actual failure - append it to the logger as last argument `logger.debug(..., failure)`. also, add some context to the message `Unable to send DM to user {} in guild {} to inform them about a scam message being blocked`

Reply:
Sure